apps/state/lib/state/stops_on_route.ex

```diff
@@ -187,6 +192,19 @@ defmodule State.StopsOnRoute do
     Enum.filter(global_order, &MapSet.member?(group_stop_id_map, &1))
   end
 
+  # additional ordered stop IDs to include during the merge
+  defp stop_order_overrides(route_id, direction_id) do
+    config = Application.get_env(:state, :stops_on_route)[:stop_order_overrides]
```

Comment:
Thoughts on putting this in code instead of config? It'd remove the need to fetch the config, the `@external_resource`, and make it easier to understand the code path.

Reply:
- The `@external_resource` was already removed, as now this happens at run time.
- Having it in configuration (like the shape overrides) limits the possible changes and makes the code easier to understand.